Support_Your_Locals/Controllers/AuthController.cs

```diff
@@ -34,12 +36,20 @@ public ActionResult SignUp(UserRegisterModel register)
                 register.AlreadyExists = exists;
                 if (!exists)
                 {
+                    new RNGCryptoServiceProvider().GetBytes(salt = new byte[16]);
```

Comment:
Maybe it would be more convenient to move encryption/decryption logic to a separate utility class?

Reply:
Maybe Identity would solve this. However, I am not yet sure if you think we should use it. Either way, I also think that creating a utility class is indeed a good solution.